spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context "validations" do
+    it "email must be unique" do
+      user = create(:user)
```

Comment:
@riverswb we can clean this up with shoulda matchers if you want:
https://github.com/thoughtbot/shoulda-matchers

E.g.,
```
it { should validate_uniqueness_of :email }
```